README.md

```diff
@@ -30,43 +30,55 @@ Nightly (main):
 
 ## Status
 
-### 1.8.1
+### v1.8.1
 
-The software is finished -- and by finished, we mean there are no show-stopping, little-children-killing bugs in it. That we know of. There are probably numerous lower-priority bugs triaged into the next point release or service pack, as well.
+The software is production-ready, capable of serving large mission-critical applications as well, and we're not aware of any fundamental bugs or missing features we deem crucial. Orchard Core continues to evolve, with each version bringing new improvements, and keeping up with the cutting-edge of .NET.
```

Comment:
```suggestion
The software is production-ready, and capable of serving large mission-critical applications as well, and we're not aware of any fundamental bugs or missing features we deem crucial. `Orchard Core` continues to evolve, with each version bringing new improvements, and keeping up with the cutting-edge of .NET.
```

Reply:
OK about the "and", but "Orchard Core" is not a technical name. That would be `OrchardCore`, but since this is  normal English text, we shouldn't use that.